packages/interbit/src/chainManagement/watchChain.js

```diff
@@ -1,4 +1,4 @@
-const _ = require('lodash')
+const { diff } = require('deep-diff')
 
```

Comment:
What is the context of this change? All of the other changes seem to have been getting Wallaby to work.

Reply:
Ah, sorry, that was for the cascading deploy changes I was working on when I realized our coverage reports were missing tests.